h2/src/docsrc/help/help.csv

```diff
@@ -855,7 +855,7 @@ work. Linked tables to the same database share one connection.
 
 READONLY - is set, the remote table may not be updated. This is enforced by H2.
 
-FETCH_SIZE - the number of rows fetched
+FETCH_SIZE - the number of rows fetched, set to ""java.sql.Statement.setFetchSize()"" method.
```

Comment:
`a hint with non-negative number of rows to fetch from the external table at once, may be ignored by the driver of external database. 0 is default and means no hint. The value is passed to ""java.sql.Statement.setFetchSize()"" method.`

Reply:
ok. Added